### ISSUE TITLE
Permission Denied when trying to create the "updater_tmp" directory

### ISSUE THREAD

**Issue by ltsdw** [bartoszek/AUR-blender-plugin-retopoflow-git#1]
While trying to enable the addon through Blender as a normal user, the addon will try to create a directory to put its json data temporarily.

![Screenshot-03-01-2024_20-29-58](https://github.com/bartoszek/AUR-blender-plugin-retopoflow-git/assets/44977415/e64e6961-3ca1-4afa-b034-fb81dffd9071)

I think the PKGBUILD may need to be updated, but I don't know if this is the best option, as the addon that was supposed to create this directory.

But as I don't see any harm in doing it manually, maybe we could also create this directory while packaging the addon like so:

```patch
--- a/PKGBUILD	2024-01-03 20:32:26.102080509 -0300
+++ b/PKGBUILD	2024-01-03 20:33:39.091693274 -0300
@@ -27,6 +27,7 @@
   cd ${srcdir}
   addons="$pkgdir/usr/share/blender/${_blender}/scripts/addons"
   install -dm755 ${addons}/${name}
+  install -dm755 ${addons}/${name}/updater_tmp
   cp -r ${name}/* ${addons}/${name}
 }
```
 
Also for the retopoflow files like `RetopoFlow_options.json`, `RetopoFlow_keymaps.json`, etc you still need root access to save keymaps, options and so on for retopoflow, but at least with the change above the addon can be enabled without root access.

**Comment by bartoszek** [comment]
Yep, this is indeed a problem... will try to patch plugin to use `$XDG_CONFIG_HOME` for the time being just drop the zip in `~/.config/blender/$version_major.$version_minor/` directory.
